### PR TITLE
Return more readable error to frontend

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: yaptide/celery/tasks.py [hunk 1]
```diff
@@ -110,6 +114,30 @@ def pymchelper_output_to_json(estimators_dict: dict) -> dict:
...
+def get_shieldhitlog(path: str) -> str:
+    """Proto version of returning errors -> returning shieldhit.log"""
+    try:
+        with open(path, 'r') as reader:
+            return reader.read()
+    except FileNotFoundError:
+        return "Shieldhitlog not found"
+
+
+def get_shield_input(path: str) -> dict:
```
Comment (grzanka) [inline yaptide/celery/tasks.py]: Rename to `simulation_input_files`,  add a docstring explaining what should be provided as input argument `path`.

File: yaptide/celery/tasks.py [hunk 2]
```diff
@@ -110,6 +114,30 @@ def pymchelper_output_to_json(estimators_dict: dict) -> dict:
...
+    """Function returning input files"""
```
Comment (grzanka) [inline yaptide/celery/tasks.py]: `files` is too enigmatic. 
```suggestion
    """Function returning a dictionary with keys being simulation input filenames and values holding the content of these files"""
```

File: yaptide/examples/call_api_example.py [hunk 3]
```diff
@@ -60,34 +58,30 @@ def run_simulation_on_backend():
...
-                    return
+                if res.status_code == 200:
+                    if data['content'].get('result'):
+                        with open(os.path.join(example_dir, 'output', 'simulation_output.json'), 'w') as writer:
+                            data_to_write = str(data['content']['result'])
+                            data_to_write = data_to_write.replace("'", "\"")
+                            writer.write(data_to_write)
+                        session.delete(http_auth_logout)
+                        return
+                    if data['content'].get('shieldhitlog'):
```
Comment (grzanka) [inline yaptide/examples/call_api_example.py]: Rename `shieldhitlog` to `logfile`

File: yaptide/celery/tasks.py [hunk 4]
```diff
@@ -132,6 +160,11 @@ def simulation_task_status(task_id: str) -> dict:
     elif task.state != 'FAILURE':
         if 'result' in task.info:
             result['content']['result'] = task.info.get('result')
+        elif 'shieldhitlog' in task.info:
+            result['content']['state'] = 'FAILURE'
+            result['content']['error'] = 'Shieldhit error'
```
Comment (grzanka) [inline yaptide/celery/tasks.py]:
```suggestion
            result['content']['error'] = 'Simulation error'
```

File: yaptide/celery/tasks.py [hunk 5]
```diff
@@ -110,6 +114,30 @@ def pymchelper_output_to_json(estimators_dict: dict) -> dict:
     return result_dict
 
 
+def get_shieldhitlog(path: str) -> str:
```
Comment (grzanka) [inline yaptide/celery/tasks.py]: Rename to `simulation_logfile`, add proper docstring

File: yaptide/celery/tasks.py (hunk 2 above)
Comment (p1003) [reply yaptide/celery/tasks.py]: Your line was to long so I changed it

File: yaptide/examples/call_api_example.py (hunk 3 above)
Comment (p1003) [reply yaptide/examples/call_api_example.py]: Done

File: yaptide/celery/tasks.py (hunk 5 above)
Comment (p1003) [reply yaptide/celery/tasks.py]: Done

File: yaptide/examples/call_api_example.py [hunk 6]
```diff
@@ -60,34 +58,30 @@ def run_simulation_on_backend():
...
-                        data_to_write = str(data['content']['result'])
-                        data_to_write = data_to_write.replace("'", "\"")
-                        writer.write(data_to_write)
-                    session.delete(http_auth_logout)
-                    return
-                if res.status_code == 500:
-                    print(data)
-                    session.delete(http_auth_logout)
-                    return
+                if res.status_code == 200:
```
Comment (grzanka) [inline yaptide/examples/call_api_example.py:66]:
```suggestion
                # the request has succeeded, we can access its contents
                if res.status_code == 200:
```

**Comment by grzanka** [comment]
@p1003 is it still draft ?

**Review by ostatni5 — DISMISSED** [review]
ok

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**